### PR TITLE
Windows pyomo solver

### LINKED ISSUE #2: Coin-CBC not available on Windows
The nominal solver used by the pyomo dispatcher is `cbc`, but that is not available through conda or otherwise on windows.

### PR DESCRIPTION
**Pull request by PaulTalbot-INL**
Coin-CBC is not readily available through conda (or otherwise) on Windows; as such, the default solver for pyomo on Windows should be one that is more commonly available (glpk).

In the future, it might be necessary to add a user-based request to determine which solver to use for pyomo.

Closes #2.

### CODE REVIEW

**Comment by PaulTalbot-INL** [comment]
Don't merge this yet; I need to use this to test the continuous integration soon.

**Comment by PaulTalbot-INL** [comment]
@abhinavg-INL this can be reviewed and merged; the repo testing will take a bit yet.

File: dependencies.xml [hunk 1]
```diff
@@ -2,36 +2,14 @@
   <main>
```
Comment (ghost) [inline dependencies.xml:2]: Can we add a header tag here `<?xml version = "number" encoding = "XYZ" ?>` for future changes in the versions? 
Can we add a description

File: src/dispatch/pyomo_dispatch.py [hunk 2]
```diff
@@ -10,6 +10,7 @@
 import time as time_mod
 from functools import partial
 from collections import defaultdict
+import platform
 
 import numpy as np
 import pandas as pd
```
Comment (ghost) [inline src/dispatch/pyomo_dispatch.py]: Why we need pandas?

Comment (ghost) [reply src/dispatch/pyomo_dispatch.py]: I don't think it is needed by this peace of code.

File: dependencies.xml (hunk 1 above)
Comment (PaulTalbot-INL) [reply dependencies.xml:2]: This file is defined by RAVEN, so I don't think that makes sense to add here, personally.

File: src/dispatch/pyomo_dispatch.py (hunk 2 above)
Comment (PaulTalbot-INL) [reply src/dispatch/pyomo_dispatch.py]: this line wasn't changed by this PR, so it's not under review; however, I have changed it.

**Comment by PaulTalbot-INL** [comment]
I added the issue and feature request templates to this PR as well.

**Review by ghost — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+##### Please attach the input file(s) that generate this error.  The simpler the input, the faster we can find the issue.
+
+
+
+----------------
+For Change Control Board: Issue Review
+----------------
+This review should occur before any development is performed as a response to this issue.
+- [ ] 1. Is it tagged with a type: defect or task?
+- [ ] 2. Is it tagged with a priority: critical, normal or minor?
```
Comment (abhinavg-INL) [inline .github/ISSUE_TEMPLATE.md:22]: minor

**Comment by ghost** [comment]
--------
Pull Request Description
--------
##### What issue does this change request address?


##### What are the significant changes in functionality due to this change request?


----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [X] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [X] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/HERON/wiki/Code-Standards) for details.
- [x] 4. Automated Tests should pass.
- [x] 5. If significant functionality is added, there must be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added, the the analytic documentation must be updated/added.
- [x] 9. If any test used as a basis for documentation examples have been changed, the associated documentation must be reviewed and assured the text matches the example.

**Comment by abhinavg-INL** [comment]
--------
Issue Description
--------
##### What did you expect to see happen?


##### What did you see instead?


##### Do you have a suggested fix for the development team?


##### Please attach the input file(s) that generate this error.  The simpler the input, the faster we can find the issue.



----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [ ] 1. Is it tagged with a type: defect or task?
- [X] 2. Is it tagged with a priority: critical, normal or minor?
- [ ] 3. If it will impact requirements or requirements tests, is it tagged with requirements?
- [ ] 4. If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [ ] 5. Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.)

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [ ] 1. If the issue is a defect, is the defect fixed?
- [ ] 2. If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [ ] 3. If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [ ] 4. If the issue is a defect, does it impact the latest release branch? If yes, is there any issue tagged with release (create if needed)?
- [ ] 5. If the issue is being closed without a pull request, has an explanation of why it is being closed been provided?

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**